### PR TITLE
Combine install doc and system requirements doc into one

### PR DESCRIPTION
**Pull request by DanRoscigno**
Please review, but **do not merge**.  I will merge a change to the nav and remove a conflicting doc in clickhouse-docs at the same time.

- combines the requirements doc and install doc in one as the requirements should be read first
- updates the URL for the deb package files
- removes duplicate info
- adds in Cloud as an option

### Changelog category (leave one):
- Documentation (changelog entry is not required)

### CODE REVIEW

File: docs/en/getting-started/install.md [hunk 1]
```diff
@@ -1,13 +1,34 @@
 ---
-sidebar_label: Installation
-sidebar_position: 1
-keywords: [clickhouse, install, installation, docs]
-description: ClickHouse can run on any Linux, FreeBSD, or Mac OS X with x86_64, AArch64, or PowerPC64LE CPU architecture.
-slug: /en/getting-started/install
-title: Installation
+sidebar_label: Install
+keywords: [clickhouse, install, getting started, quick start]
+slug: /en/install
```
Comment (Felixoid) [inline docs/en/getting-started/install.md:4]: does it mean that it won't be available by the old path?

File: docs/en/getting-started/install.md [hunk 2]
```diff
@@ -1,13 +1,34 @@
...
+- **[ClickHouse Cloud](https://clickhouse.cloud/):** the official ClickHouse as a service, - built by, maintained, and supported by the creators of ClickHouse
+- **Self-managed ClickHouse:** ClickHouse can run on any Linux, FreeBSD, or Mac OS X with x86_64, AArch64, or PowerPC64LE CPU architecture
+
+## ClickHouse Cloud
+
+The quickest and easiest way to get up and running with ClickHouse is to create a new service in [ClickHouse Cloud](https://clickhouse.cloud/):
+
+<div class="eighty-percent">
+
+![Create a ClickHouse Cloud service](@site/docs/en/_snippets/images/createservice1.png)
```
Comment (Felixoid) [inline docs/en/getting-started/install.md:20]: It's w/ `@site` in one place, w/o in other. And it used to be relative links, and now it's absolute. Not sure, why the absolute are better?

File: docs/en/getting-started/install.md [hunk 3]
```diff
@@ -19,7 +40,54 @@ $ grep -q sse4_2 /proc/cpuinfo && echo "SSE 4.2 supported" || echo "SSE 4.2 not
 
 To run ClickHouse on processors that do not support SSE 4.2 or have AArch64 or PowerPC64LE architecture, you should [build ClickHouse from sources](#from-sources) with proper configuration adjustments.
 
-## Available Installation Options {#available-installation-options}
```
Comment (Felixoid) [inline docs/en/getting-started/install.md:22]: I think it's better to preserve links

File: docs/en/getting-started/install.md (hunk 2 above)
Comment (DanRoscigno) [reply docs/en/getting-started/install.md:20]: `@site` gets used for images and include files.  They all convert to relative links.  We are having problems with `./foo.md` and `../../foo.md` links working on initial load of a page, but 404ing no reloads.  Using the `/docs/...` fixes this.  As I receive issues about 404's I am fixing the individual docs, and as I edit docs I am fixing them.

File: docs/en/getting-started/install.md (hunk 3 above)
Comment (DanRoscigno) [reply docs/en/getting-started/install.md:22]: ok

File: docs/en/getting-started/install.md [hunk 4]
```diff
@@ -58,9 +126,9 @@ clickhouse-client # or "clickhouse-client --password" if you set up a password.
 
 </details>
 
-You can replace `stable` with `lts` to use different [release kinds](../faq/operations/production.md) based on your needs.
+You can replace `stable` with `lts` to use different [release kinds](/docs/en/faq/operations/production.md) based on your needs.
 
-You can also download and install packages manually from [here](https://packages.clickhouse.com/deb/pool/stable).
+You can also download and install packages manually from [here](https://packages.clickhouse.com/deb/pool/main/c/).
```
[the remark this answers is not in the document]
Comment (DanRoscigno) [reply docs/en/getting-started/install.md:131]: sure thing!

File: docs/en/getting-started/install.md (hunk 1 above)
Comment (DanRoscigno) [reply docs/en/getting-started/install.md:4]: the PR in the clickhouse-docs repo adds the redirects, so install.md, and requirements.md links both go the same place.  No old links will break.

File: docs/en/getting-started/install.md (hunk 3 above)
Comment (DanRoscigno) [reply docs/en/getting-started/install.md:22]: added back in, does not show in this diff, but if you view the whole file it is there.

File: docs/en/getting-started/install.md (hunk 1 above)
Comment (DanRoscigno) [reply docs/en/getting-started/install.md:4]: this is the other PR: https://github.com/ClickHouse/clickhouse-docs/pull/438

File: docs/en/getting-started/install.md [hunk 5]
```diff
@@ -19,7 +40,56 @@ $ grep -q sse4_2 /proc/cpuinfo && echo "SSE 4.2 supported" || echo "SSE 4.2 not
...
+
+The network bandwidth is critical for processing distributed queries with a large amount of intermediate data. Besides, network speed affects replication processes.
+
+### Software {#software}
+
+ClickHouse is developed primarily for the Linux family of operating systems. The recommended Linux distribution is Ubuntu. The `tzdata` package should be installed in the system.
+
+## Self-Managed Install
+
+### Available Installation Options {#available-installation-options}
```
Comment (Felixoid) [inline docs/en/getting-started/install.md:92]: not sure if would work

`## Self-Managed Install {#self-managed-install} {#available-installation-options`

Would it?

Comment (DanRoscigno) [reply docs/en/getting-started/install.md:92]: I tried, it uses the last `{}` in the line as the anchor and makes the others part of the title:
image

**Comment by DanRoscigno** [comment]
closes https://github.com/ClickHouse/product/issues/562

File: docs/en/getting-started/install.md (hunk 5 above)
Comment (Felixoid) [reply docs/en/getting-started/install.md:92]: I see. But now we have an empty `Available Installation Options`

Comment (DanRoscigno) [reply docs/en/getting-started/install.md:92]: yes.  If we remove it and someone uses a link that has that `#available...` then they end up at the top of the page, and they see the choices of Debs, RPMs, tar files etc. in the right side nav.  If we add the line back in then we have an extra H2, which I don't mind.  Whichever you prefer.

Comment (Felixoid) [reply docs/en/getting-started/install.md:92]: TBH, I prefer a hidden one, for sure. If there would be a trick to place it in the text below, that would be the best..

If no, H2 looks a bit more organic, WDYT?

Comment (DanRoscigno) [reply docs/en/getting-started/install.md]: Agreed, adjusted to H2.

**Review by Felixoid — APPROVED** [review]

**Pull request merged**